functions-app/poison-replay-completed-checks/function.json

```diff
@@ -0,0 +1,5 @@
+{
+  "disabled": false,
+  "bindings": [
+  ]
```

Comment:
please add timer trigger input binding to a schedule of something arbitrary like once a year in 2030.
Also set disabled to false.
Then we can play this manually from the portal when necessary, as you suggested.